app/controllers/exam_templates_controller.rb

```diff
@@ -138,6 +138,36 @@ def view_logs
                                  .includes(:user)
   end
 
+  def assign_errors
+    @assignment = Assignment.find(params[:assignment_id])
+    @error_files = []
+    Dir.foreach(File.join(
+      MarkusConfigurator.markus_exam_template_dir,
+      @assignment.short_identifier, 'error'
+    )) { |file|
+      @error_files << file unless file =~ /^\.\.?$/
+    }
+    @error_files = @error_files.sort do |a, b|
```

Comment:
Hmm. The comparison function you're using looks like the default, so I would think that just `@error_files.sort` is fine.